core/src/main/scala/org/apache/spark/internal/config/History.scala

```diff
@@ -29,6 +29,14 @@ private[spark] object History {
     .stringConf
```

Comment:
not related to this PR, but `EVENT_LOG_DIR` doesn't seem a great name to me for this property, as we have a `spark.eventLog.dir`. Shall we update it to `HISTORY_LOG_DIR` or something similar in order to disambiguate?

Reply:
I'm fine with the name `HISTORY_LOG_DIR`. Let me update it.
cc @HyukjinKwon @srowen @dongjoon-hyun @vanzin

Reply:
Yep. `HISTORY_LOG_DIR` sounds good to me, too.